### ISSUE TITLE
LNK2019 unresolved external symbol in VS2019

### ISSUE THREAD

**Issue by AlexFundorin** [cnjinhao/nana#442]
I've added "include" folder from nana repository into my project - https://i.imgur.com/c0jGQXt.png
Added necessary includes - https://i.imgur.com/LgX6Z03.png
Added this simple code - https://i.imgur.com/L7ja5jy.png
Yet, I'm getting this three (more, with other example code) LNK2019 errors:
**Error	LNK2019	unresolved external symbol "public: __cdecl nana::msgbox::msgbox(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (??0msgbox@nana@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) referenced in function "private: ...**
How do I fix these errors?
I do not want to use nana as a static library, just as a simple "include".

**Comment by qPCR4vir** [comment]
And where is the compiled from .cpp nana?
And just curious, what is the reason you don't whant static linking? What  do you want instead of static linking?

**Comment by AlexFundorin** [comment]
I was able to successfully compile my project with nana. It was quite unusual that the library can't work as a regular set of include files (.h and .cpp), but only as a linked static library.

**Comment by qPCR4vir** [comment]
Hi! Donwload the sources, or just cloning with git (.h and .cpp files) and compiling is the recomended way to use nana. How are you getting nana? Why as static library?

**Comment by AlexFundorin** [comment]
At first, I've added "include" folder contents from the cloned repo to my project.
Yet, I was getting "unresolved external symbol" error upon compilation.
Then, I've compiled nana as a static library from the nana vs2017 solution and linked it to my project.
What is the preferred way of using nana? 
Instruction here is also recommending to use it as a static library.
https://github.com/qPCR4vir/nana-docs/wiki/Installation
